cypress/obj/clients/clientCreatePage.js

```diff
@@ -0,0 +1,23 @@
+class ClientCreatePage {
+    getFormOrganization() {
+        return cy.get('[id="clientDetailsForm.organization"]');
+    }
+
+    getFormFirstName() {
+        return cy.get('[id="clientDetailsForm.contacts_firstName_0"]');
+    }
+
+    getFormLastName() {
+        return cy.get('[id="clientDetailsForm.contacts_lastName_0"]');
+    }
+
+    getFormEmail() {
+        return cy.get('[id="clientDetailsForm.contacts_email_0"]');
+    }
+
+    getFormSubmit() {
```

Comment:
I believe this is a button, right? To keep consistent, let's add `...Button()` at the end so that we know it's a button we're interacting with :) I see you've done that in other places already 👍